HACKING.md

```diff
@@ -18,23 +18,52 @@ dnf install -y oci-kvm-hook # if on traditional
 ```
 
 We will use `oc cluster up` to set up a local cluster for testing. To do
-this, simply obtain the OpenShift binary from
-[the latest release](https://github.com/openshift/origin/releases/latest)
-(though note that I'm currently still using the binaries from v3.9, so
-pick that release if you hit issues).
+this, obtain the OpenShift binary from
+[the v3.9.0 release](https://github.com/openshift/origin/releases/tag/v3.9.0).
+`openshift-origin-client-tools` is enough for `oc cluster up`.
```

Comment:
not sure how I feel about telling people specifically to use 3.9. I kind of like what jonathan had where he says to use the latest but noting what version worked for him.

If we are going to hard require a version I think it whould be 3.6.1, what is in centos CI.